### ISSUE TITLE
Show statistics on the last week of scans

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-androidapp#3582]
We could have a little statistics section on the last week of scans (or of a given list) 

- [ ] pie chart repartition of Eco-Scores
- [ ] pie chart repartition of Nutri-Scores
- [ ] pie chart repartition of NOVA
- [ ] Sum of all the carbon footprints of the day/week/year https://github.com/openfoodfacts/openfoodfacts-server/issues/4503

**Comment by teolemon** [comment]
https://github.com/openfoodfacts/openfoodfacts-server/issues/4503

**Comment by zukunft** [comment]
Yes, the carbon footprints of the day/week/year in CO2-eq would be really be nice.
